src/main/webapp/cdn/blockly/generators/propc/abvolts.js

```diff
@@ -32,7 +32,7 @@ Blockly.Blocks.ab_volt_in = {
         this.setColour(colorPalette.getColor('io'));
         this.appendDummyInput()
             .appendField("ADC channel")
-            .appendField(new Blockly.FieldDropdown([["0", "0"], ["1", "1"], ["2", "2"], ["3", "3"]]), "CHANNEL");
+            .appendField(new Blockly.FieldDropdown([["0", "0"], ["1", "1"], ["2", "2"], ["3", "3"]]), "CHANNEL")
             .appendField("read (0-5V) in volt-100ths")
```

Comment:
Okay -I promise to be more careful :)

*Matthew Matz  |  STEM/Robotics Educator*
Parallax Inc.  |  Direct: 916-625-3019  |  www.parallax.com  |  @M
atzElectronics <http://twitter.com/MatzElectronics>

On Wed, Aug 31, 2016 at 9:07 AM, michel-cf <notifications@github.com> wrote:

> In src/main/webapp/cdn/blockly/generators/propc/abvolts.js
> <https://github.com/parallaxinc/BlocklyProp/pull/653#discussion_r77017607>
> :
>
> > @@ -32,7 +32,7 @@ Blockly.Blocks.ab_volt_in = {
> >          this.setColour(colorPalette.getColor('io'));
> >          this.appendDummyInput()
> >              .appendField("ADC channel")
> > -            .appendField(new Blockly.FieldDropdown([["0", "0"], ["1", "1"], ["2", "2"], ["3", "3"]]), "CHANNEL");
> > +            .appendField(new Blockly.FieldDropdown([["0", "0"], ["1", "1"], ["2", "2"], ["3", "3"]]), "CHANNEL")
> >              .appendField("read (0-5V) in volt-100ths")
>
> The last line from the statement is still missing it's ';'
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/parallaxinc/BlocklyProp/pull/653/files/59d200693a557e0142272a2c45bfc22b5e907e1e#r77017607>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AS0quB_qE1hhzLNwhoZoh2QnKNxHYV-bks5qlaawgaJpZM4Jxxq8>
> .
>